lib/cisco_node_utils/interface.rb

```diff
@@ -74,8 +74,6 @@ def access_vlan
 
     def access_vlan=(vlan)
       config_set('interface', 'access_vlan', @name, vlan)
-    rescue Cisco::CliError => e
-      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
```

Comment:
Let's not remove them until we have equivalent or better messaging from CliError. 
IMO the test effort from changing CliError is outside the scope of this PR so just open a US to tackle this down the road.